api/v1beta1/verticadb_types.go

```diff
@@ -317,7 +326,7 @@ type CommunalStorage struct {
 	Endpoint string `json:"endpoint"`
```

Comment:
This should have a dependency on initPolicy too.  It isn't needed for ScheduleOnly.